tests/src/test/java/de/sovity/edc/e2e/UiApiWrapperTest.java

```diff
@@ -591,6 +594,7 @@ void checkIdAvailability(E2eScenario scenario, @Provider EdcClient providerClien
         assertThat(posContractDefinitionResponse.getAvailable()).isTrue();
     }
 
+    @DisabledOnGithub
```

Comment:
💀

Reply:
I still have a ticket open for this.

Reply:
https://github.com/sovity/edc-ce/issues/870